commerce-add-ons/packages/product-feeds/installation.md

```diff
@@ -0,0 +1,24 @@
+[nuget_url]: https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/
+
+## Quick start
+1. Install the package from [NuGet][nuget_url] &middot; [![NuGet](https://img.shields.io/nuget/v/Umbraco.Commerce.ProductFeeds.svg?style=modern&label=nuget)][nuget_url]
+
+2. Locate where you [register the dependencies](https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies) `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()` to add this plugin to your website.
+```csharp
+        umbracoBuilder.AddUmbracoCommerce(ucBuilder => {
+            ucBuilder.AddCommerceProductFeeds(); // add this line
+        }
+```
+
+3. Open your store's setting page in the backoffice 
+	
+4. Click on `Product Feed` section
+![product feed list page](./media/product-feed-list-page.png)
+
+5. Click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.
+![feed setting page](./media/feed-setting-page.png)
+
+6. After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.
+![open feed link](./media/open-feed-link.png)
+`Google Merchant Center Feed` sample:
+![google merchant center feed](./media/google-merchant-center-feed.png)
```

Comment:
Adding missing content from other 2 packages + adding new line before images so they can show in new line and not inline with content
```suggestion

---
description: >-
Detailed instructions on how to install and configure Product Feeds into your Umbraco Commerce implementation.
---

The Product Feeds package can be installed directly into your project's code base using NuGet packages. Below you can find some steps on installing the package using NuGet and getting started with implementing the Product Feeds into your Umbraco Commerce store.

# Getting started

1. Install the package from [NuGet](https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/)

2. Locate where you [register the dependencies](https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies) `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()` to add this plugin to your website.

        umbracoBuilder.AddUmbracoCommerce(ucBuilder => {
            ucBuilder.AddCommerceProductFeeds(); // add this line
        }

3. Open your store's setting page in the backoffice.
	
4. Click on `Product Feed` section.

![product feed list page](./media/product-feed-list-page.png)

5. Click on `Create Product Feed` button and fill in the feed settings. Mandatory fields are marked with a red asterisk (*). You can add more product data by adding new mappings in `Property And Node Mapping` section.

![feed setting page](./media/feed-setting-page.png)

6. After saving the feed setting, a link to access the feed will show up under `Feed URL Segment` field and at the bottom of the page.

![open feed link](./media/open-feed-link.png)

`Google Merchant Center Feed` sample:

![google merchant center feed](./media/google-merchant-center-feed.png)

## Upgrading

{% hint style="info" %}
Before upgrading, it is always advisable to take a complete backup of your site/database.
{% endhint %}

The Checkout package uses a combination of database migrations and a manual install dashboard for both installs and upgrades. Upgrading is generally a case of installing the latest version over the existing package and running through the installation steps.

```